synth.py

```diff
@@ -159,11 +159,35 @@
 os.system('rm -rf dev/system-test/fixtures dev/system-test/install.ts')
 
 node.install()
-node.fix()
 os.chdir("dev")
 node.compile_protos()
 os.chdir("protos")
 os.unlink('protos.js')
 os.unlink('protos.d.ts')
 subprocess.run('./update.sh', shell=True)
 os.chdir("../../")
+
+# Copy types into types/
+os.system("cp build/src/v1/firestore*.d.ts types/v1")
```

Comment:
Did you verify that this works if there is no "build" folder prior to this run?

Reply:
Since we run `npm.install()` before this step, the `build/` folder should be generated already.